src/ffpuppet/bootstrapper.py

```diff
@@ -18,39 +16,56 @@
 
 
 class Bootstrapper:  # pylint: disable=missing-docstring
-    BUF_SIZE = 4096  # receive buffer size
-    POLL_WAIT: float = 1  # duration of initial blocking socket operations
-    PORT_MAX = 0xFFFF  # bootstrap range
-    PORT_MIN = 0x3000  # bootstrap range
```

Comment:
Why not keep `PORT_MIN` instead of copying `BLOCKED_PORTS`?

Reply:
Using the minimum blocks way more ports than needed, 11264 if we use 0x3000 only 22 are blocked.